ONSdigital/rm-collection-exercise-service
# Add status column to events table

# Motivation and Context


# What has changed




# How to test?
- Create events in your environment (create collection exercise and set mps, golive, etc against it)
- Update your config to have `deprecateAction` to true.
- Deploy to your environment.  You should notice the events you created earlier should have NOT_SET.
- Create new events, these should have `SCHEDULED` status.

Note, while it's possible to have collection exercises with events with both old (NOT_SET) and new (SCHEDULED, etc) states.  When we do the switchover, there will be a bit of work to change all the not sets to SCHEDULED (if they're in the future and need to be done) and PROCESSED (if they're old events).  Theres a card in the backlog to handle this.  The NOT_SET status is a temporary measure to allow us to switch over.





# Links
